Update coroutine.md

gxlct008 opened the pull request:
修改部分语句使其更通顺，修改方法名的大小写保持文中一致。

gxlct008 committed:
Update coroutine.md

修改部分语句使其更通顺，修改方法名的大小写保持文中一致。

deminy commented on zh-ch/coroutine.md:
```diff
@@ -10,34 +10,34 @@
+## 什么是Channel
 
-`channel`可以理解为消息队列，只不过是协程间的消息队列，多个协程通过`push`和`pop`操作生产消息和消费消息，用来协程之间的通讯。需要注意的是`channel`是没法跨进程的，只能一个`Swoole`进程里的协程间通讯，最典型的应用是[连接池](/coroutine/conn_pool)和[并发调用](/coroutine/multi_call)。
+`Channel`可以理解为消息队列，只不过是协程间的消息队列，多个协程通过`push`和`pop`操作队列中的生产消息和消费消息，用来完成协程之间的通讯。需要注意的是`Channel`是没法跨进程的，只能一个`Swoole`进程里的协程间通讯，最典型的应用是[连接池](/coroutine/conn_pool)和[并发调用](/coroutine/multi_call)。
```
`用来完成协程之间的通讯` 这个描述更具体些，但我感觉还是不够准确？ @sy-records

gxlct008 replied:
“完成” 改成 “达成” 如何？
另外，感觉还是把 “通讯” 改成 “通信” 好些。

修改一：
“用来完成协程之间的通讯”  改成 “以完成协程之间的通信”。

修改二：
“用来完成协程之间的通讯”  改成 “以达成协程之间的通信”。

sy-records replied:
```suggestion
`Channel`可以理解为消息队列，只不过是协程间的消息队列，多个协程通过`push`和`pop`操作队列中的生产消息和消费消息，用来发送或者接收数据进行协程之间的通讯。需要注意的是`Channel`是没法跨进程的，只能一个`Swoole`进程里的协程间通讯，最典型的应用是[连接池](/coroutine/conn_pool)和[并发调用](/coroutine/multi_call)。
```

deminy commented on zh-ch/coroutine.md:
```diff
@@ -10,34 +10,34 @@
+决定到底让CPU执行哪个协程的代码的决断过程就是`协程调度`，`Swoole`的调度策略又是怎么样的呢？
 
-- 首先，在执行某个协程代码的过程中发现这行代码遇到了`Co::sleep()`或者产生了网络`IO`，例如`MySQL->query()`，这肯定是一个耗时的过程，`Swoole`就会把这个Mysql连接的Fd放到[EventLoop](/learn?id=什么是eventloop)中。
+- 首先，在执行某个协程代码的过程中发现这行代码遇到了`Co::sleep()`或者产生了网络`IO`，例如`MySQL->query()`，这肯定是一个耗时的过程，`Swoole`就会把这个MySQL连接的Fd放到[EventLoop](/learn?id=什么是eventloop)中。
```
`Fd`改成小写（`fd`）为宜。这是码农常用的一个缩写，但不是专有名词。

gxlct008 committed:
Update zh-ch/coroutine.md

`Channel`可以理解为消息队列，只不过是协程间的消息队列，多个协程通过`push`和`pop`操作队列中的生产消息和消费消息，发送或接收数据进行通信。需要注意的是`Channel`是没法跨进程的，只能同一个`Swoole`进程里的协程间通信，最典型的应用是[连接池](/coroutine/conn_pool)和[并发调用](/coroutine/multi_call)。

Co-authored-by: 沈唁 <52o@qq52o.cn>

sy-records commented on zh-ch/coroutine.md:
```diff
@@ -10,34 +10,34 @@
 ## 父子协程优先级
 
-优先执行子协程(即`go()`里面的逻辑)，直到发生协程`yield`(co::sleep处)，然后[协程调度](/coroutine?id=协程调度)到外层协程
+优先执行子协程(即`go()`里面的逻辑)，直到发生协程`yield`(Co::sleep()处)，然后[协程调度](/coroutine?id=协程调度)到外层协程。
```
这块需要看下渲染后的结果，()怎么显示的

gxlct008 replied:
同样是这篇文章，后面的段落里 “优先执行子协程 (即 go() 里面的逻辑)” 有例子。@sy-records

deminy replied:
有区别的。go()那里用了反引号(**\`go()\`**)的，你添加的这个没有。这应该就是为什么 @sy-records 有顾虑的原因。因为渲染出来后就不一定是`Co::sleep()`了。
